js/WebView.ios.js

```diff
@@ -442,16 +447,8 @@ class WebView extends React.Component {
   }
 }
 
-const RCTWebView = requireNativeComponent(
-  'RCTWebView',
-  WebView,
-  WebView.extraNativeComponentConfig,
```

Comment:
I don't know where these were being specified but you can probably delete these too.